src/test/java/ca/bc/gov/open/cdds/CourtControllerTests.java

```diff
@@ -76,7 +76,7 @@ public void getDigitalDisplayCourtList() throws IOException {
         a.setCourtClassCd("A");
         a.setFileNumberTxt("A");
         a.setCounselFullNm("A");
-        clr.setAppearance(Collections.singletonList(a));
+        // clr.setAppearance(Collections.singletonList(a));
```

Comment:
@jianmingtu, @KFloodBCGov, As discussed, this is the only change I had to make that was a little concerning and unexpected.  It appears the Appearance class has completely changed and does not contain a getter or setter for Appearance at all anymore.

@jianmingtu, Is looking into this.

Reply:
@jianmingtu, I have a fix for this coming.

Reply:
Issue resolved.